spec/models/template_update_spec.rb

```diff
@@ -28,4 +28,14 @@
         .to 'moomin'
     end
   end
+
+  describe '#enqueue!' do
+    # rubocop:disable RSpec/MessageSpies
```

Comment:
Stubbing out `Hyrax::Workflow::TemplateNotification` seems at risk of testing implementation rather than the desired behavior.

Is the end goal for `Hyrax::MessengerService` to get a `#deliver` for each admin? Would spying that work better?

```ruby
let(:number_of_admins) { 2000 # except some real way of knowing the number of admins }

it do
  allow(Hyrax::MessengerService).to receive(:deliver)

  batchable.enqueue!

  expect(Hyrax::MessengerService)
    .to have_received(:deliver)
    .exactly(number_of_admins)
    .times
end
```

Reply:
It would be really cool to have a fake implementation of `Hyrax::MessengerService` that we could use as a more complete spy. This seems like something to add to `Hyrax::Spec`, though I think it would require a refactor in Hyrax to avoid the hard coded module name for the service.